src/azul/project/hca/transformers.py

```diff
@@ -179,7 +179,7 @@ def create_documents(self,
             file.ancestors(visitor)
             # Assign the contents to the ES doc
             contents = dict(specimens=_specimen_dict(visitor.specimens),
-                            files=list(visitor.files.values()),
+                            files=[visitor._file_dict(file)],
```

Comment:
Please make `_file_dict` a top-level function (like `_specimen_dict` and `_project_dict`). This should be the first commit in your branch.